test/libsolidity/syntaxTests/denominations/invalid_denomination_address.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+	address a = 0x11111122222333334444455555666667777788888 wei;
```

Comment:
I wanted to specifically cover the address type. I wasn't actually sure if it would even work. It's a separate type and in the code needs to be handled explicitly.

Reply:
Makes sense.